code/modules/mob/living/living.dm

```diff
@@ -703,6 +703,10 @@
 		var/obj/mecha/M = loc
 		loc_temp =  M.return_temperature()
 
+	else if(istype(loc, /obj/spacepod))
```

Comment:
Uh, this isn't quite what I meant.

all of get temp should become:

```
/mob/living/proc/get_temperature(datum/gas_mixture/environment)
    var/loc_temp = T0C
    if(isobj(loc))
        var/obj/oloc = loc
        var/obj_temp =oloc.return_temperature()
        if(obj_temp != null)
            loc_temp = obj_temp
    if(isspaceturf(get_turf(src)))
        var/turf/heat_turf = get_turf(src)
        loc_temp = heat_turf.temperature
    else
        loc_temp = environment.temperature
    return loc_temp
```

This means transit_tube_pods and cryotubes need a ```return_temperature()``` override (all it would do is the code it does for them here, just moved there)